.travis.yml

```diff
@@ -2,8 +2,8 @@ language: ruby
 rvm:
   - 2.3.0
 
-services:
-  - postgresql
+addons:
+  postgresql: "9.4"
```

Comment:
Let's make sure we double check with devops about the version of pg we'll be using before merging, just in case.